File: Swift-C++-IR-Comparison/test/closures/Swift/ClosuresSortExpressive.swift
@main
struct MyMain {

    static func main() {
        let numbers = [42, 1, 47, 22, 23, 24]
        var reversedNumbers = numbers.sorted(by: backward)
    }

}

func backward(_ s1: Int, _ s2: Int) -> Bool {
    return s1 > s2
}

File: Swift-C++-Testsuite/AutoTranspiledTests/try-0/ClosuresSortExpressive.cpp

#include      <vector>
#include      <algorithm>
#include      <iostream>

using namespace std;

bool backward(int s1, int s2) {
    return s1 > s2;
}

int main(){
    vector<int> numbers = {42, 1, 47, 22, 23, 24};
    sort(numbers.begin(), numbers.end(), backward);
    return 0;
}